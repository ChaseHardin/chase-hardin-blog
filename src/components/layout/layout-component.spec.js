import React from "react"
import {configure, shallow} from 'enzyme'
import Adapter from 'enzyme-adapter-react-16';

import Layout from "./layout-component"

configure({ adapter: new Adapter() });

describe("Layout", () => {
  it('should render prop message', () => {
    const props = {
      children: 'anything'
    }

    const component = shallow(<Layout {...props}/>);

    expect(component.text()).toEqual(props.children)
  })
})
