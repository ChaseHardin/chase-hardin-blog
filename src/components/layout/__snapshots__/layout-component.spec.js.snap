// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
<div
  style={
    Object {
      "margin": "3rem auto",
      "maxWidth": 650,
      "padding": "0 1rem",
    }
  }
>
  &lt;h1&gt;Hello&lt;/h1&gt;
</div>
`;
